app/src/main/java/com/example/playlistmaker/App.kt

```diff
@@ -0,0 +1,39 @@
+package com.example.playlistmaker
+
+import android.app.Application
+import android.content.SharedPreferences
+import androidx.appcompat.app.AppCompatDelegate
+
+
+
+
+class App : Application() {
+    companion object{
+        const val PLAYLISTMAKER = "playlist_maker_shared_preferences"
+        const val NIGHT_MODE_KEY = "night_mode_key"
+    }
+    var darkTheme = false
+    private lateinit var sharedPreferences: SharedPreferences
+    override fun onCreate() {
+        super.onCreate()
+        sharedPreferences = getSharedPreferences(PLAYLISTMAKER, MODE_PRIVATE)
+        darkTheme = sharedPreferences.getBoolean(NIGHT_MODE_KEY, false)
+        if (darkTheme) {
+            switchTheme(true)
+        }
```

Comment:
```suggestion
        switchTheme(darkTheme)
```
Лучше написать так, чтобы не было бага, когда пользователь открывает приложение в тёмной теме, заходит в настройки, а там выключенный бегунок